profile/BBmri-observation-bodyweight.StructureDefinition.json

```diff
@@ -0,0 +1,52 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "https://fhir.bbmri.de/StructureDefinition/BBmri-observation-bodyweight",
+  "name": "BBmri-observation-bodyweight",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Observation",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/bodyweight",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Observation.subject",
+        "path": "Observation.subject",
+        "type": [
+          {
+            "code": "Reference",
+            "targetProfile": [
+              "http://fhir.germanbiobanknode.de/fhir/StructureDefinition/GbaPatient"
```

Comment:
Patient with old URL?

Reply:
Yes, this branch uses the bbmri.de URL only for newly created resources. The old URLs will be fixed in an extra branch that will also unify the naming conventions.